native-advanced/server.js

```diff
@@ -9,6 +9,7 @@ const routing = {
   '/user': user,
   '/user/name': () => user.name,
   '/user/age': () => user.age,
+  // client is passed but never used
```

Comment:
That's ok, because `(client, par)` is a contract of the handler, like `(error, data)` in callback-last/error-first contract, so first argument can be ignored but other handler may use it.